x/ibc/spec/01_concepts.md

```diff
@@ -38,7 +38,13 @@ given by the chain's chainID, and the epoch height given by the Tendermint block
 and resets its block-height, it is responsible for updating its chain-id to increment the epoch number.
 IBC Tendermint clients then verifies the epoch number against their `ChainId` and treat the `EpochHeight` as the Tendermint block-height.
 
-TODO: Explain how to structure chain-id to make epoch-number parsable
+Tendermint chains wishing to use epochs to maintain persistent IBC connections even across height-resetting upgrades must format their chain-ids
+in the following manner: `{chainID}-epoch-{EpochNumber}`. On any height-resetting upgrade, the chainID **MUST** be updated with a higher epoch number
+than the previous value.
+
+Ex:
+Before upgrade ChainID: `gaia-epoch-3`
+After upgrade ChainID: `gaia-epoch-4`
```

Comment:
most of the chains use `<chainID>-<number>`, should we use that instead of adding the epoch in between?

Reply:
Decision on Slack:

```
Aditya:
Is there a particular format people had in mind for encoding epochs in chain-id. I was thinking about either having a loose approach or a strict approach.
The loose approach would be doing something like splitting the chain id by "-"  and then just trying to interpret the last element of the split string as a number and using that as the epoch number.
This would work even if the chain-id is just a number as seems to be the case in ethermint
Or I could go with a stricter approach of having something like `{chain-id}-epoch-{epochNumber}` and only accepting chain-ids in that format

Chris:
I think this & defaulting to an epoch of 0 is preferable
epochs should only be used by chains which specifically want this kind of safe upgrade path
```

The format we currently enforce forces chains to EXPLICITY opt-in to this upgrade path, rather than accidentally. Tho I am open to changing this default. cc: @cwgoes

Reply:
I think `{chainID}-{epoch}` is alright, but we should enforce that there is only one `-` instead of just splitting the string and trying to parse.